esmvaltool/utils/cmorizers/obs/cmorize_obs_ESACCI-OC.py

```diff
@@ -0,0 +1,169 @@
+# pylint: disable=invalid-name
+"""ESMValTool CMORizer for ESACCI-OC data.
+
+Tier
+
+Source
+   ftp://oceancolour.org/occci-v3.1/geographic/netcdf/monthly/chlor_a/
+
+Last access
+   20190227
+
+Download and processing instructions
+   Put all files under a single directory (no subdirectories with years)
+   in ${RAWOBS}/Tier2/ESACCI-OC
+
+Modification history
+   20190227-A_lova_to: written.
+
+"""
+
+import logging
+import os
+import glob
+import xarray as xr
+
+import iris
+
+from .utilities import (constant_metadata, fix_coords, fix_var_metadata,
+                        read_cmor_config, save_variable, set_global_atts)
+
+logger = logging.getLogger(__name__)
+
+# read in CMOR configuration
+CFG = read_cmor_config('ESACCI-OC.yml')
+
+
+def _fix_data(cube, var):
+    """Specific data fixes for different variables."""
+    logger.info("Fixing data ...")
+    with constant_metadata(cube):
+        if var == 'chl':
+            cube *= 1.e-06
+    return cube
+
+
+# pylint: disable=unused-argument
+def _fix_auxcoord(cube, field, filename):
+    """Add depth auxiliary coordinate for CMIP5 standard."""
+    if not cube.coords('depth'):
+        depth = 1.
+        depth_coord = iris.coords.AuxCoord(
+            depth,
+            standard_name='depth',
+            long_name='depth',
+            var_name='depth',
+            units='m',
+            attributes={'positive': 'down'})
+        cube.add_aux_coord(depth_coord)
+        cube.coordinates = 'depth'
+
+
+def extract_variable(var_info, raw_info, out_dir, attrs):
+    """Extract to all vars."""
+    var = var_info.short_name
+    cubes = iris.load(raw_info['file'], callback=_fix_auxcoord)
+    rawvar = raw_info['name']
+
+    for cube in cubes:
+        if cube.var_name == rawvar:
+            fix_var_metadata(cube, var_info)
+            fix_coords(cube)
+            _fix_data(cube, var)
+            set_global_atts(cube, attrs)
+            thefile = save_variable(
```

Comment:
```suggestion
            save_variable(
```
it looks like you're not using `thefile` for anything

Reply:
removed! it was a leftover from the previous implementation with xarray.